subsys/fs/nvs/nvs.c

```diff
@@ -277,6 +277,48 @@ static int nvs_ate_cmp_const(const struct nvs_ate *entry, uint8_t value)
 	return 0;
 }
 
+/* nvs_ate_valid validates an ate:
+ *     return 1 if crc8 and offset valid,
+ *            0 otherwise
+ */
+static int nvs_ate_valid(struct nvs_fs *fs, const struct nvs_ate *entry)
+{
+	size_t ate_size;
+
+	ate_size = nvs_al_size(fs, sizeof(struct nvs_ate));
+
+	if ((nvs_ate_crc8_check(entry)) ||
```

Comment:
According to `.checkpatch.conf` you can have now 100 chars per line so ou do not have to break these `if`s

Reply:
restored the original line.